index.html

```diff
@@ -24,8 +24,8 @@ <h2 class="quiz-card__question">
           <img src="./bookmark_clear.svg" width="30" height="30" alt="" />
         </div>
         <details>
-          <summary>Show answer</summary>
-          <div class="quiz-card__answer">Tomorrow.</div>
+          <summary class="quiz-card__summary">Show answer</summary>
+          <div class="quiz-card__answer details">Tomorrow.</div>
```

Comment:
This could be a headline or paragraph instead of a div.

Reply:
Changed into `p`-Elements.